### PR TITLE
fix(console): remove connector id and prevent text overflow

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

For add connector modal: remove connector id and prevent text overflow

## Testing

image

### CODE REVIEW

File: packages/console/src/pages/Connectors/components/CreateForm/index.module.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@
       max-width: 276px;
       width: 276px;
       height: 96px;
+      overflow: hidden;
```
Comment (charIeszhao) [inline packages/console/src/pages/Connectors/components/CreateForm/index.module.scss:13]: What is this for?

Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/CreateForm/index.module.scss:13]: Prevent overflow when there are too many lines

Comment (simeng-li) [reply packages/console/src/pages/Connectors/components/CreateForm/index.module.scss:13]: multi-line ellipsis?

```

display: -webkit-box;
text-overflow: ellipsis;
 -webkit-line-clamp: 2;
 -webkit-box-orient: vertical;
 overflow: hidden;
```

Comment (charIeszhao) [reply packages/console/src/pages/Connectors/components/CreateForm/index.module.scss:13]: Yes, and we can even make this a mixin and call it `clamp(n)`

Comment (gao-sun) [reply packages/console/src/pages/Connectors/components/CreateForm/index.module.scss:13]: FYI `packages/console/src/components/Tooltip/index.module.scss`

image

Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/CreateForm/index.module.scss]: updated with a new mixin

**Review by gao-sun — APPROVED** [review]

**Pull request merged**